src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -29,7 +29,7 @@
 from ._client_factory import cf_acr_registries
 from ._constants import get_managed_sku
 from ._utils import get_registry_by_name, ResourceNotFound
-
+from ._constants import ACR_CHECK_HEALTH_MSG
```

Comment:
The idea was to recommend `check_health`  command in future if other commands fail. Therefore, I thought it would be better to make it a constant since it might be used across other places in future.

Reply:
recommend moving it to check_health.py file.